### ISSUE TITLE
Chunk loading peripheral causes game crash

### ISSUE THREAD

**Issue by quinchs** [Seniorendi/AdvancedPeripherals#80]
# Summary

When placing a chunky turtle the game crashes.
### Versions:

- Forge version: 36.1.18
- AdvancedPeripherals version: 0.6r

### Crashlog
```
[25May2021 14:23:17.599] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: Encountered an unexpected exception
java.lang.NoSuchMethodError: net.minecraft.world.server.TicketManager.registerTicking(Lnet/minecraft/world/server/TicketType;Lnet/minecraft/util/math/ChunkPos;ILjava/lang/Object;)V
	at net.minecraft.world.server.ServerChunkProvider.registerTickingTicket(ServerChunkProvider.java:433) ~[?:?]
	at net.minecraftforge.common.world.ForgeChunkManager.forceChunk(ForgeChunkManager.java:172) ~[?:?]
	at net.minecraftforge.common.world.ForgeChunkManager.reinstatePersistentChunks(ForgeChunkManager.java:251) ~[?:?]
	at net.minecraftforge.common.world.ForgeChunkManager.reinstatePersistentChunks(ForgeChunkManager.java:211) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213186_a(MinecraftServer.java:485) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240800_l__(MinecraftServer.java:310) ~[?:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:86) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:621) [?:?]
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) [?:?]
	at net.minecraft.server.MinecraftServer$$Lambda$7662/458901277.run(Unknown Source) [?:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_51]
```

**Comment by Seniorendi** [comment]
[advancedperipherals-0.6.2.1b.zip](https://github.com/Seniorendi/AdvancedPeripherals/files/6542278/advancedperipherals-0.6.2.1b.zip)
Hey, could you try if this version fixes the issue? Unzip the file before using it.

**Comment by Seniorendi** [comment]
Fixed in 0.6.4b
